### DISCUSSION TITLE
why the value of " s_track_upper_restart" all nan all nan?

### DISCUSSION THREAD

**Discussion by YiyingWang-00** [WAM2layers/WAM2layers#210] [Q&A] [answered]
The backtrack.py has been successfully run, but the result is s_track_upper_restart all nan, and the tagged_precipitation is 0, what cause this problem？(The processed data is true.)Thank you!

**Comment by YiyingWang-00** [comment]
How can I caculate the precipitation recyclying ratio by the output data?

**Reply by ruudvdent** [reply]
Dear @2021310510041 , thanks for reaching out. The basics can naturally be found in several papers. But perhaps your question is more specific about how to do it from this output data. Some old calculations can be found in [ruudvdent/WAM2layersPython](https://github.com/ruudvdent/WAM2layersPython), but they still need to be added to version 3. If you want to contribute to the code, please check the [documentation](https://wam2layers.readthedocs.io/en/latest). Since we are a community, please consider introducing yourself and your research in [Users of WAM2layers](https://github.com/WAM2layers/WAM2layers/discussions/categories/users-of-wam2layers)

**Comment by YiyingWang-00** [comment]
If I don't need to tracke individual events, how can I set the event_start_date and event_end_date in config file?

**Reply by ruudvdent** [reply]
Then start and end date would be equal to the start and end date of your entire data series

**Reply by YiyingWang-00** [reply]
Thanks for your help! I can run successfully now!

**Comment by Peter9192** [comment, marked as answer]
Dear @2021310510041,
Welcome to the WAM2Layers, and thanks for reaching out. We appreciate your feedback and suggestions, but without more context it is difficult to diagnose your problem. You may want to look through the open issues and the GitHub discussions to see if others have reported similar issues.

**Comment by DuBaolong** [comment]
> The backtrack.py has been successfully run, but the result is s_track_upper_restart all nan, and the tagged_precipitation is 0, what cause this problem？(The processed data is true.)Thank you!

i face the same problem recently. did you have solved the problem?

**Comment by ssomani10** [comment]
May have and idea of what they are communicating about as I tried the sample data and was obtaining a nan.  See attached screenshot from my Mac and I had the same error when I tried using Ubuntu from a virtual machine.  Please note I didn't make any changes to the YAML file.

<img width="1214" alt="Backtrack_Error" src="https://user-images.githubusercontent.com/8172915/232181918-6a5759fc-d27b-4e1f-a20c-2ca4fbce5824.png">

**Comment by YiyingWang-00** [comment]
> > The backtrack.py has been successfully run, but the result is s_track_upper_restart all nan, and the tagged_precipitation is 0, what cause this problem？(The processed data is true.)Thank you!
> 
> i face the same problem recently. did you have solved the problem?

Have you solved the problem? I couldn't solve yet, but I think it may caused by config file.

**Reply by ruudvdent** [reply]
Did you check whether the data after pre-processing looked fine? Could you re-check this with the updates from the latest release?

**Comment by ssomani10** [comment]
I have not solved it yet.  Let's discuss this further in the discussion forum, as the developers want this the place for bug discussions.
